app/wasm/stargate/whitelist.go

```diff
@@ -0,0 +1,134 @@
+	setWhitelistedQuery("/cosmos.gov.v1beta1.Query/Vote", &govtypes.QueryVoteResponse{})
+
+	// slashing
+	setWhitelistedQuery("/cosmos.slashing.v1beta1.Query/Params", &slashingtypes.QueryParamsResponse{})
+	setWhitelistedQuery("/cosmos.slashing.v1beta1.Query/SigningInfo", &slashingtypes.QuerySigningInfoResponse{})
+
+	// staking
+	setWhitelistedQuery("/cosmos.staking.v1beta1.Query/Delegation", &stakingtypes.QueryDelegationResponse{})
+	setWhitelistedQuery("/cosmos.staking.v1beta1.Query/Params", &stakingtypes.QueryParamsResponse{})
+	setWhitelistedQuery("/cosmos.staking.v1beta1.Query/Validator", &stakingtypes.QueryValidatorResponse{})
+
+	// umee native module queries
+
+	// ugov
+	setWhitelistedQuery("/umee.ugov.v1.Query/MinGasPrice", &ugovtypes.QueryMinGasPriceResponse{})
+
+	// leverage
+	setWhitelistedQuery("/umee.leverage.v1.Query/Params", &ltypes.QueryParamsResponse{})
+	setWhitelistedQuery("/umee.leverage.v1.Query/RegisteredTokens", &ltypes.QueryRegisteredTokensResponse{})
+	setWhitelistedQuery("/umee.leverage.v1.Query/MarketSummary", &ltypes.QueryMarketSummaryResponse{})
+	setWhitelistedQuery("/umee.leverage.v1.Query/AccountBalances", &ltypes.QueryAccountBalancesResponse{})
+	setWhitelistedQuery("/umee.leverage.v1.Query/AccountSummary", &ltypes.QueryAccountSummaryResponse{})
+	setWhitelistedQuery("/umee.leverage.v1.Query/LiquidationTargets", &ltypes.QueryLiquidationTargetsResponse{})
+	setWhitelistedQuery("/umee.leverage.v1.Query/BadDebts", &ltypes.QueryBadDebtsResponse{})
+	setWhitelistedQuery("/umee.leverage.v1.Query/MaxWithdraw", &ltypes.QueryMaxWithdrawResponse{})
+	setWhitelistedQuery("/umee.leverage.v1.Query/MaxBorrow", &ltypes.QueryMaxBorrowResponse{})
+
+	// oracle
+	setWhitelistedQuery("/umee.oracle.v1.Query/ExchangeRates", &otypes.QueryExchangeRatesResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/ActiveExchangeRates", &otypes.QueryActiveExchangeRatesResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/FeederDelegation", &otypes.QueryFeederDelegationResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/MissCounter", &otypes.QueryMissCounterResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/SlashWindow", &otypes.QuerySlashWindowResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/AggregatePrevote", &otypes.QueryAggregatePrevoteResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/AggregatePrevotes", &otypes.QueryAggregatePrevotesResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/AggregateVote", &otypes.QueryAggregateVoteResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/AggregateVotes", &otypes.QueryAggregateVotesResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/Params", &otypes.QueryParamsResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/Medians", &otypes.QueryMediansResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/MedianDeviations", &otypes.QueryMedianDeviationsResponse{})
+	setWhitelistedQuery("/umee.oracle.v1.Query/AvgPrice", &otypes.QueryAvgPriceResponse{})
```

Comment:
nit: I think it could be nice to have something like `setWhitelistedQuery(oracleBaseQueryPath+"myQuery", ...)` with `const oracleBaseQueryPath = "/umee.oracle.v1.Query/"`. 
It would help to spot potential typo make PR reviews easier. WDYT?

Reply:
I think `"/" + proto.MessageName(msg)` will work as well. Each query is a ProtoMessage

Reply:
No , both are different 

here register path is `/umee.oracle.v1.Query/ActiveExchangeRates` but when I run with `proto.MessageName(msg)` the result is `umee.oracle.v1.QueryActiveExchangeRates`

Reply:
https://github.com/umee-network/umee/pull/2069/files#r1213691385
We will keep like this `/umee.oracle.v1.Query/Params` bcause it is readable

Reply:
Yes it is quite readable but feels like that if`/umee.oralce.v1/` arrive it will be hard to spot between the rest